packages/@react-aria/list/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "@react-aria/list",
+  "version": "3.0.0-alpha.1",
+  "private": true,
```

Comment:
I forget, should I directly modify this or will the version/private be updated upon publish?